http.kafka.cqrs/src/main/java/io/aklivity/example/cqrs/processor/SuccessfulCommandProcessor.java

```diff
@@ -0,0 +1,113 @@
+package io.aklivity.example.cqrs.processor;
+
+import java.util.UUID;
+
+import org.apache.kafka.common.header.Header;
+import org.apache.kafka.common.header.Headers;
+import org.apache.kafka.common.header.internals.RecordHeader;
+import org.apache.kafka.common.header.internals.RecordHeaders;
+import org.apache.kafka.streams.processor.api.Processor;
+import org.apache.kafka.streams.processor.api.ProcessorContext;
+import org.apache.kafka.streams.processor.api.ProcessorSupplier;
+import org.apache.kafka.streams.processor.api.Record;
+import org.apache.kafka.streams.state.KeyValueStore;
+
+import io.aklivity.example.cqrs.model.Command;
+import io.aklivity.example.cqrs.model.CreateTaskCommand;
+import io.aklivity.example.cqrs.model.DeleteTaskCommand;
+import io.aklivity.example.cqrs.model.Task;
+import io.aklivity.example.cqrs.model.UpdateTaskCommand;
+
+public class SuccessfulCommandProcessor implements ProcessorSupplier<String, Command, String, Object>
```

Comment:
Rename to `AcceptCommandProcessorSupplier`, and move anonymous inner class to named inner class called `AcceptCommandProcessor`.